Formula/orientdb.rb

```diff
@@ -48,23 +43,36 @@ def post_install
     (var/"log/orientdb").mkpath
     touch "#{var}/log/orientdb/orientdb.err"
     touch "#{var}/log/orientdb/orientdb.log"
+
+    # set the server's root user  password to changeme
+    ENV["ORIENTDB_ROOT_PASSWORD"] = "orientdb"
+    system "/usr/bin/nohup  #{libexec}/bin/server.sh &"
```

Comment:
Can `orientdb start` or similar be used here instead? Alternatively, use `fork` and `exec` rather than `nohup`.